compiler/rustc_target/src/spec/armebv7r_none_eabi.rs

```diff
@@ -20,6 +20,8 @@ pub fn target() -> Target {
             panic_strategy: PanicStrategy::Abort,
             max_atomic_width: Some(32),
             emit_debug_gdb_scripts: false,
+            // GCC and Clang default to 8 for arm-none here
+            c_enum_min_bits: 8,
```

Comment:
It might be a good idea to have some deduplication between all the `arm*-none-eabi(hf)` target specs, but it doesn't have to happen in this PR.